packages/lib/src/data/messages_controller.ts

```diff
@@ -204,6 +204,108 @@ export class LogMessageEvent {
 	}
 }
 
+export class SystemMetrics {
```

Comment:
None of the getter are tested, this is a simple one to add as you only need to call `fromJson` and then check against known good values.